### ISSUE TITLE
wrong HTML for image from other module

### ISSUE THREAD

**Issue by wols** [Mogztter/antora-site-generator-lunr#2]
```
.npm-global/lib64
├── @antora/cli@2.1.1
├── @antora/site-generator-default@2.1.1
└── antora-site-generator-lunr@0.1.1
```

```
// modules/ROOT/pages/index.adoc
image::Doc:diagram/doc.png[Diagram from module Doc]
```
**Wrong** HTML by `antora-site-generator-lunr@0.1.1`:
```
<img src="Doc:diagram/doc.png" alt="Diagram from module Doc">
```
**Correct** HTML by `@antora/site-generator-default@2.1.1`:
```
<img src="Doc/_images/diagram/doc.png" alt="Diagram from module Doc">
```

**Comment by Mogztter** [comment]
Not sure maybe it has been fixed in Antora 2.1+.
We are still using Antora 2.0 as a dependency...

**Comment by Mogztter** [comment]
Yes it's a new feature in Antora 2.1: https://docs.antora.org/antora/2.1/whats-new/#reference-images-from-other-modules-components-and-versions-using-antoras-resource-id

I've upgraded to Antora 2.1.1 in fe01116c367ede7b2482de66d52acbee47a046f0
